libs/expo/betterangels/src/lib/screens/Client/Docs/UploadModal/Section.tsx

```diff
@@ -0,0 +1,57 @@
+import { ChevronLeftIcon } from '@monorepo/expo/shared/icons';
```

Comment:
Can we change `Section` to something more descriptive? e.g. `UploadScreen`

Reply:
changed to UploadSection